integration_tests/cellarfees_test.go

```diff
@@ -0,0 +1,418 @@
+package integration_tests
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"fmt"
+	"log"
+	"math/big"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	disttypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	ethereumtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	auctiontypes "github.com/peggyjv/sommelier/v4/x/auction/types"
+	cellarfeestypes "github.com/peggyjv/sommelier/v4/x/cellarfees/types"
+	corktypes "github.com/peggyjv/sommelier/v4/x/cork/types"
+)
+
+const ALPHA_FEE_DENOM string = "gravity0x4C4a2f8c81640e47606d3fd77B353E87Ba015584"
```

Comment:
The linter is angry with the naming pattern here. Just confirming, when we deploy these contracts we get consistent addresses every time so that we can actually hardcode them in this manner?

Reply:
We do